02-visualization.Rmd

```diff
@@ -439,8 +417,7 @@ early_january_weather <- weather %>%
 Let's create a time series plot of the hourly temperatures saved in the `early_january_weather` data frame by using `geom_line()` to create a linegraph\index{ggplot2!geom\_line()}, instead of using `geom_point()` like we used previously to create scatterplots:
 
 ```{r hourlytemp, fig.cap="Hourly temperature in Newark for January 1-15, 2013."}
-ggplot(data = early_january_weather, 
-       mapping = aes(x = time_hour, y = temp)) +
+ggplot(data = early_january_weather, mapping = aes(x = time_hour, y = temp)) +
```

Comment:
I worry that this might hurt the readability of the code without the newline?

Reply:
```suggestion
ggplot(data = early_january_weather,
       mapping = aes(x = time_hour, y = temp)) +
```